dev/src/Command/AddComponentCommand.php

```diff
@@ -111,17 +123,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product homepage? ')
-        );
-
-        $productDocumentation = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product documentation URL? ')
-        );
+        $version = basename(dirname($proto));
+        // yaml file contains documentation_uri and
+        // it is available at the same location as proto file
+        $libraryName = strtolower($new->componentName);
+        $yamlFileName = $libraryName . '_' . $version . '.yaml';
+        $yamlFilePath = dirname($proto) . '/' . $yamlFileName;
+        $googleApisDir = explode('/google/', $yamlFilePath)[0];
+        $productDocumentation = null;
+        if (file_exists($yamlFilePath)) {
+            $yamlFileContent = file_get_contents($yamlFilePath);
+            preg_match('/(?i)^.*\bdocumentation_uri\b.*$/m', $yamlFileContent, $matches);
```

Comment:
we have a YAML component which parses yaml files. We should use that instead. e.g.:

```php
$yamlFileContents = Yaml::parseFile($yamlFilePath);
$productDocumentation = $yamlFileContents['publishing']['documentation_uri'] ?? null;
```